src/avram/query_builder.cr

```diff
@@ -308,6 +308,12 @@ class Avram::QueryBuilder
     block.call(self)
   end
 
+  # Clears the last conjunction
+  # e.g. users.age = $1 AND -> users.age = $1
+  def clear_conjunction
+    @wheres.last.conjunction = Avram::Where::Conjunction::None unless @wheres.empty?
```

Comment:
This was needed because the last where might `"("` and it was generating `WHERE ( AND users.age...`